lib/constants.js

```diff
@@ -152,7 +152,7 @@ const ContractCode = {
   StakingProducts: 'SP',
   TokenController: 'TC',
   // TokenData: 'TD',
-  YieldTokenIncidents: 'CG',
+  // YieldTokenIncidents: 'CG',
```

Comment:
just rename

Reply:
Same here, should we just comment to keep consistency?

Reply:
yep, let's keep it consistent